build-logic-commons/gradle-plugin/src/main/kotlin/gradlebuild.code-quality.gradle.kts

```diff
@@ -66,12 +66,6 @@ val errorproneExtension = project.extensions.create<ErrorProneProjectExtension>(
     nullawayEnabled.convention(false)
 }
 
-nullaway {
```

Comment:
could move this to central config to avoid extra coupling. 

Its kind of the same as checked in:

- https://github.com/junit-team/junit-framework/pull/5006
  - https://github.com/junit-team/junit-framework/pull/5006/files#r2419903879